src/version.h

```diff
@@ -4,15 +4,16 @@
 #define VERSION_MAJOR               0
 #define VERSION_MINOR               5
 #define VERSION_REVISION            3
+#define VERSION_PATCH               2
```

Comment:
We don't do patch, as revision == patch

Reply:
Patch is so I can track what beta testers have been using, this can always be 0 for release, if that's OK?